README.md

```diff
@@ -351,10 +351,7 @@ When submitting code, please make every effort to follow existing conventions an
 * Paweł Urban (pawel.urban@polidea.com)
 
 ## Contributors, thank you!
-* Michał Zieliński (michal.zielinski@polidea.com)
-* Fracturedpsyche (https://github.com/fracturedpsyche)
-* Andrea Pregnolato (https://github.com/pregno)
-* Matthieu Vachon (https://github.com/maoueh) - custom operations, yay!
+http://bit.ly/rxandroidble-contributors
```

Comment:
Maybe
`## [Contributors](https://github.com/Polidea/RxAndroidBle/graphs/contributors), thank you!`?